bin/check-process.rb

```diff
@@ -261,7 +270,10 @@ def run
     procs.reject! { |p| p[:command] =~ /#{config[:exclude_pat]}/ } if config[:exclude_pat]
     procs.reject! { |p| p[:command] !~ /#{config[:cmd_pat]}/ } if config[:cmd_pat]
     procs.select! { |p| p[:vsz].to_f > config[:vsz] } if config[:vsz]
+    # Ensure RSS is over this value
     procs.select! { |p| p[:rss].to_f > config[:rss] } if config[:rss]
+    # Ensure RSS is under this value
```

Comment:
Oh geez this check is confusing...I am not sure what to do with this check as I want to burn it to the ground and re-write it as multiple checks its trying to do way too many things that have way too many use cases and therefore edge cases. Basically what we were missing was to use `-W 1 || --warn-over 1, -C || --crit-over 1` as by default it checks if its under rather than over.

Here is an example:
```
$ ./bin/check-process.rb -p google --resident-set-size 368381 -c 1
CheckProcess CRITICAL: Found 5 matching processes; cmd /google/; rss > 368381
```

```
$ ./bin/check-process.rb -p google --resident-set-size 368381
CheckProcess OK: Found 6 matching processes; cmd /google/; rss > 368381
```